src/main/scala/org/broadinstitute/dsde/firecloud/utils/HttpClientUtils.scala

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.dsde.firecloud.utils
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorRefFactory
+import akka.http.scaladsl.HttpExt
+import akka.http.scaladsl.coding.Gzip
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.{HttpEncodings, `Accept-Encoding`}
+import akka.http.scaladsl.unmarshalling.{Unmarshal, Unmarshaller}
+import akka.stream.Materializer
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.FireCloudExceptionWithErrorReport
+import org.broadinstitute.dsde.rawls.RawlsExceptionWithErrorReport
+import org.broadinstitute.dsde.rawls.model.{ErrorReport, ErrorReportSource}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Success
+
+/**
+  * Created by rtitle on 7/7/17.
+  */
+trait HttpClientUtils extends LazyLogging {
```

Comment:
Originally lifted from rawls but as with DsdeHttpDAO, I don't think we actually need it in orch. i have one more place in HttpGoogleServicesDAO to rip this out of and then i think this will disappear

Reply:
I just removed DsdeHttpDAO and HttpClientUtils. RestJsonClient is used everywhere now